packages/lodestar-cli/package.json

```diff
@@ -9,7 +9,14 @@
   },
   "homepage": "https://github.com/ChainSafe/lodestar#readme",
   "main": "lib/index.js",
-  "files": [],
+  "files": [
+    "lib/**/*.d.ts",
+    "lib/**/*.js",
+    "lib/**/*.js.map"
```

Comment:
include source files or this map file is useless :smile:

Reply:
In case the CLI throws an error, won't the source maps be useful to trace the stack to the TS source?

Reply:
yes, but it doesn't work if there aren't any source files, should add `src/**/*.ts` line here. 
That said, sourcemaps only work if you have node12.12 and add flag --enable-source-maps

Reply:
In other CLIs I've develop source files work fine without adding the source `*.ts` files. But I initially suggested this change for consistency since this `"files"` is the one currently used in all other Lodestar monorepo packages.

Reply:
> In other CLIs I've developed source files work fine without adding the source *.ts files

You are right, it will display correct path, we can leave it like this. Those stack traces are links in my terminal and opens files when clicked. I doubt many people are using it so it's totally fine not send it.

Reply:
Summoning @wemeetagain so we can have a working CLI in NPM :heart: